test/regressions/index.js

```diff
@@ -133,10 +133,10 @@ vrtest.before(() => {
       },
       timeout: 20000,
       active: () => {
-        resolve('active');
+        resolve('webfontloader: active');
       },
       inactive: () => {
-        resolve('inactive');
+        reject(new Error('webfontloader: inactive'));
```

Comment:
I don't know yet was wrong with the font awesome flaky regression test. I suspect at network fetch fail issue (not a concurrency issue). I hope a reject can help me identify a fix.

Reply:
It doesn't seem to help. A topic for another pull request.